layer/USAGE_android.md

```diff
@@ -26,8 +26,8 @@ It may also be possible to grant external storage permissions to
 an installed application through the device Settings.
 
 **Failure to enable the write external storage permission** can cause
-the layer to crash if it tries and fails to create a capture file on
-the external storage device.
+the layer to return VK_ERROR_INITIALIZATION_FAILED from its vkCreateInstance
```

Comment:
VK_ERROR_INITIALIZATION_FAILED => `VK_LAYER_INITIALIZATION_FAILED`

I'm also thinking vkCreateInstance => `vkCreateInstance`, but I'm less bullish about that.